virttest/tests/unattended_install.py

```diff
@@ -399,6 +399,16 @@ def answer_kickstart(self, answer_path):
             raise ValueError("Unexpected installation medium %s" % self.url)
         contents = re.sub(dummy_medium_re, content, contents)
 
+        dummy_guest_name_re = r'\bRH_GUEST_NAME\b'
```

Comment:
Is there are reason for the `RH_` prefix? The guest could be any distro AFAIK.

Reply:
@pevogam Thanks for your review. These piece of code is for internal RHEL build(not released version), other distributions should not need this parameter, it will skip this step by default since Line 403 would be False. Since there is no similar requirement from other distributions so far, I suppose we could adjust it if there is in the future.